sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/Customizations/HDInsightJobManagementClient.Config.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Azure.HDInsight.Job
     using Microsoft.Rest;
     using System;
 
-    public partial class HDInsightJobManagementClient : ServiceClient<HDInsightJobManagementClient>, IHDInsightJobManagementClient
+    public partial class HDInsightJobClient : ServiceClient<HDInsightJobClient>, IHDInsightJobClient
```

Comment:
This is a breaking change and requires a package version bump.

Reply:
> This is a breaking change and requires a package version bump.

Hi @AlexGhiondea ,we just checked in the source code 2 years ago into github repo, but we hasn't published sdk based the source code. This means there is no sdk with version "3.0.0-preview.1". Do we still need to change the package's version?

Reply:
If that is the case, then no.